source/docs/articles/sites/migrate/wordpress-site-networks.md

```diff
@@ -0,0 +1,136 @@
+
+You can use Git to import your existing code and commit history. If you don’t have a Git version controlled codebase, the following will still work.
+
+1. Navigate to your existing site's code directory in a local terminal. If your existing code is not version controlled with Git, run:
+
+ ```bash
+ git init
+ git add .
+ git commit -m "initial commit"
+ ```
+2. From the Dev environment of the Site Dashboard, set the site's [connection mode](/docs/articles/getting-started/#interact-with-your-code) to Git.
+3. Copy the SSH URL for the site repository, found in the [clone command](/docs/articles/local/starting-with-git/#step-2-copy-the-git-clone-command). **Do not copy `git clone` or the site name.** The URL should look similar to the following:
+
+ ```bash
+ ssh://codeserver.dev.{site-id}@codeserver.dev.{site-id}.drush.in:2222/~/repository.git
+ ```
+
+4. Use Git to pull in the upstream's code (which may have Pantheon-specific optimizations) to your existing site's codebase, replacing `<ssh_url>` with the SSH URL copied in step 3:
+
+ ```bash
+ git pull --no-rebase --squash -Xtheirs <ssh_url> master
+ ```  
+
+ Will yield:  
+ ```bash
+ Squash commit -- not updating HEAD  
+ Automatic merge went well; stopped before committing as requested
+ ```
+Preserve any logic necessary in the original `wp-config.php` and `.gitignore` files.
+5. Add Pantheon as a remote destination, replacing `<ssh_url>` with the SSH URL copied in Step 3:
+
+ ```bash
+ git remote add pantheon <ssh_url>
+ ```
+
+6. Run `git add` and `commit` to prepare the Pantheon core merge for pushing to the repository:
+ ```bash
+ git add -A
+ git commit -m "Adding Pantheon core files"
+ ```
+7. Push your newly merged codebase up to your Pantheon site repository:
+
+ ```bash
+ git push pantheon master --force
+ ```
+
+8. Go to the Code tab of your Dev environment on the Site Dashboard. The most recent commit adds Pantheon's core files. This process preserves the commit history for site's already utilizing version control and once pushed your pre-existing commits will be visible on the Dashboard.
+
+## Files
+
+**Files** - `/wp-content/uploads` is a symlink to the `~/files` directory. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc. For information on highly populated directories, see [Platform Considerations](/docs/articles/sites/platform-considerations/#highly-populated-directories).
+
+Files are stored separately from the site's code. Larger file structures can fail in the Dashboard import due to sheer volume. It's best to use a utility such as an SFTP client or rsync. The biggest issue is having the transfer stopped due to connectivity issues. To handle that scenario, try this handy bash script:  

```

Comment:
The two paragraphs above are a bit awkward. I think it could be worth explicitly mentioning what the uploads directory is used for in WordPress (any content uploaded through the dashboard), and that the uploads directory is a shared filesystem / the only writeable directory on project. If your existing WordPress Site Network stored site uploads in another directory, then you'll need to reconcile this as a part of the import process.